gecko/glue/nsAppRunner.cpp

```diff
@@ -0,0 +1,14 @@
+/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+extern "C" {
+bool
+XRE_IsContentProcess()
+{
+  //return XRE_GetProcessType() == GeckoProcessType_Content;
```

Comment:
We probably want to pretend to be in single process Firefox, i.e. we'd want to pretend we're GeckoProcessType_Default. Otherwise some code may think it needs to call over to the main process for various services etc. This will do for now.